meta/collection-requirements.yml

```diff
@@ -1,2 +1,4 @@
 collections:
   - name: community.general
+  - name: community.mysql
+  - name: community.postgresql
```

Comment:
add `- name: ansible.posix` to fix this error:
```
ERROR! couldn't resolve module/action 'ansible.posix.firewalld'. This often indicates a misspelling, missing collection, or incorrect module path.
```

Reply:
oops, this little fella got lost on the way of merge ;)